docs/modules/ROOT/pages/spring-cloud-circuitbreaker.adoc

```diff
@@ -107,25 +107,73 @@ Customizer.once(circuitBreaker -> {
 [[interface-clients]]
 == Spring Interface Clients Support
 
-We provide support for Spring Interface Clients integration through the following configurers:
+Spring Cloud provides support for Spring Interface Clients integration through the following configurers:
 
 - `CircuitBreakerRestClientHttpServiceGroupConfigurer`
 - `CircuitBreakerWebClientHttpServiceGroupConfigurer`
 
 These configurers enable CircuitBreaker support for https://docs.spring.io/spring-framework/reference/7.0-SNAPSHOT/integration/rest-clients.html#rest-http-interface-group-config[Spring Interface Client Groups].
 
-When fallback class names are configured using:
-
-- `spring.cloud.http.client.service.group.[groupName].fallback-class-names` (for a specific group), or
-- `spring.cloud.http.client.service.fallback-class-names` (as a default for all groups),
-
-a CircuitBreaker adapter is added to the respective group:
+When fallback classes are configured using the `@HttpServiceFallbackAnnotation`,
+CircuitBreaker adapter decorators are added:
 - `CircuitBreakerAdapterDecorator` is used with `RestClient`
 - `ReactiveCircuitBreakerAdapterDecorator` is used with `WebClient`
 
+[NOTE]
+====
+You can disable CircuitBreaker integration for HTTP service clients by setting the appropriate property:
+
+- For blocking (`RestClient`) clients: `spring.cloud.circuitbreaker.http-services.enabled=false`
```

Comment:
Is this necessary?  Why would they add the annotations if they didn't want to use them?

Reply:
Ah I see.  This is fine for a milestone.  I wonder if we should add some kind of marker class to `spring-cloud-circuitbreaker` to indicate its on the classpath and only run things if it is, thoughts?

Reply:
Right. Makes sense. Will create a separate PR.